src/Butler/Auth.hs

```diff
@@ -24,3 +25,19 @@ htmlMain xfiles title body = do
 
         with body_ [class_ "font-mono cursor-default bg-stone-100 h-screen"] do
             body
+
+guestAppDisplay :: App -> DisplayApplication
```

Comment:
It seems like `guestAppDisplay` just sets the title of `publicDisplayApp`. So could we just add it as an argument to the existing helper and remove the hardcoded "Standalone GUI", e.g.:

```haskell
publicDisplayApp :: Text -> DisplayApplication
```

Since App can spawn other apps, and because "DisplayApplication" should work for a list of app, we shouldn't assume single app argument.

Reply:
And because such title is probably hardcoded, it might be good to use such newtype for documentation purpose:

```haskell
newtype PageTitle = PageTitle Text
  deriving newtype (IsString)

publicDisplayApp :: PageTitle -> DisplayApplication
publicDisplayApp (PageTitle title) = ...
```

… and caller would use overloaded string: `publicDisplayApp "My App!"`

Reply:
Ok, last change implements your suggestion + also add a Maybe appDesc